examples/auth/app/auth/mutations/forgotPassword.test.ts

```diff
@@ -8,15 +8,15 @@ beforeEach(async () => {
 })
 
 const generatedToken = "plain-token"
-jest.mock("@blitzjs/core/server", () => ({
-  ...jest.requireActual("@blitzjs/core/server")!,
+jest.mock("blitz", () => ({
+  ...jest.requireActual<object>("blitz")!,
```

Comment:
Thanks @Roesh!

You have to actually revert this change as it breaks example tests here in the monorepo. Maybe a way to fix it, but needs to be `@blitzjs/core/server` for now

Reply:
Ah, got it. Let me revert that